src/http/fetch_request.ts

```diff
@@ -132,7 +132,8 @@ export class FetchRequest {
 
   private async allowRequestToBeIntercepted(fetchOptions: RequestInit) {
     const requestInterception = new Promise(resolve => this.resolveRequestPromise = resolve)
-    const event = dispatch("turbo:before-fetch-request", { cancelable: true, detail: { fetchOptions, resume: this.resolveRequestPromise } })
+    const fetchUrl = this.url.href
+    const event = dispatch("turbo:before-fetch-request", { cancelable: true, detail: { fetchOptions, url: fetchUrl, resume: this.resolveRequestPromise } })
```

Comment:
Maybe just pass the value inline? The `fetchUrl` variable seems redundant here.

```suggestion
    const event = dispatch("turbo:before-fetch-request", { cancelable: true, detail: { fetchOptions, url: this.url.href, resume: this.resolveRequestPromise } })
```

Reply:
Sure, fixed :)